tilemap/tileset-wall.tres

```diff
@@ -51,9 +51,6 @@ points = PoolVector2Array( 0, 8, 16, 8, 16, 16, 0, 16 )
 [sub_resource type="ConvexPolygonShape2D" id=16]
 points = PoolVector2Array( 16, 16, 0, 16, 0, 0, 16, 0 )
 
-[sub_resource type="ConvexPolygonShape2D" id=17]
```

Comment:
What does this remove?

Reply:
no idea what this removed. i just adjusted the size of the tiles to match what we discussed yesterday. i reckon when the import happened it just assumed this shape for the tile?

Reply:
Ah yes that can be :+1: